static/app/views/issueList/actions/index.tsx

```diff
@@ -593,7 +593,6 @@ const SortDropdownMargin = styled('div')`
 const AnimatedHeaderItemsContainer = styled(motion.div)`
   display: flex;
   align-items: center;
-  line-height: 1.6;
```

Comment:
afaict this makes no difference in the old design, but it does align all columns after Issue correctly